### PR TITLE
Dygraph.dateString_: shows milliseconds if any.

### PR DESCRIPTION
**Pull request by pierriko**
1st commit add a feature to Dygraph.dateString_: shows milliseconds if any.
2nd commit fixes a typo in dateAxisLabelFormatter.

### CODE REVIEW

**Comment by danvk** [comment]
Thanks for this! If you could add a unit test that shows the new behavior, I'd be happy to merge.

**Comment by pierriko** [comment]
Great ! Let me know if the test is OK for you.

File: tests/labelsDateMilliseconds.html [hunk 1]
```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>UTC date labels</title>
```
Comment (danvk) [inline tests/labelsDateMilliseconds.html]: Update title.

File: tests/labelsDateMilliseconds.html [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    <script type="text/javascript" src="../dist/dygraph.js"></script>
+  </head>
+  <body>
+
+    <h2>Milliseconds display in date and time labels</h2>
+
+    <p>This shows how milliseconds are displayed when present.</p>
+
+    <div id="div_loc" style="width:600px; height:200px;"></div>
+    <div id="div_utc" style="width:600px; height:200px;"></div>
```
Comment (danvk) [inline tests/labelsDateMilliseconds.html]: update these names

File: tests/labelsDateMilliseconds.html [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+        return a;
+      }
+      gloc = new Dygraph(
+                   document.getElementById("div_loc"),
+                   data(2009, 6, 25, 14, 34, 56, 654),
+                   {
+                     labels: ['time with ms', 'random']
+                   }
+                 );
+      gutc = new Dygraph(
```
Comment (danvk) [inline tests/labelsDateMilliseconds.html]: also these

File: DEVELOP.md [hunk 4]
```diff
@@ -6,6 +6,10 @@ This is a step-by-step guide explaining how to do it.
 
 ### How-to
 
+To install dependencies, run
```
Comment (danvk) [inline DEVELOP.md]: thanks :)

**Comment by danvk** [comment]
Great, thanks for the fix & tests!

**Pull request merged**